ikura/chain/runtimes/gondatsu/src/lib.rs

```diff
@@ -120,7 +120,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("blobchain-kusama"),
```

Comment:
I really think we might be able to change this (it has to do with the native runtime in particular) but I don't see much urgency to do so.